source/js/validation.js

```diff
@@ -0,0 +1,47 @@
+var fieldValidation, validation;
```

Comment:
It might be nice if these weren't global but everything's working so I'd rather let someone refine this in a future PR if it's important to them. 👍